bin/kafka-server-stop.sh

```diff
@@ -14,7 +14,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 SIGNAL=${SIGNAL:-TERM}
-PIDS=$(ps ax | grep -i 'kafka\.Kafka' | grep java | grep -v grep | awk '{print $1}')
+
+if [ -z $JOBNAME ];then
```

Comment:
Can we move this `if` block inside the `if [[ $(uname -s) == "OS/390" ]] ; then` block?